op-node/node/node.go

```diff
@@ -430,7 +430,7 @@ func (n *OpNode) initL2(ctx context.Context, cfg *Config) error {
 		n.safeDB = safedb.Disabled
 	}
 	n.l2Driver = driver.NewDriver(n.eventSys, n.eventDrain, &cfg.Driver, &cfg.Rollup, n.l2Source, n.l1Source,
-		n.supervisor, n.beacon, n, n, n.log, n.metrics, cfg.ConfigPersistence, n.safeDB, &cfg.Sync, sequencerConductor, altDA)
+		n.beacon, n, n, n.log, n.metrics, cfg.ConfigPersistence, n.safeDB, &cfg.Sync, sequencerConductor, altDA)
```

Comment:
(because I am merging and resolving conflicts on this line anyway, I'll be adopting my suggestion. Feel free to argue your way, happy to hear more about your style preferences)

Reply:
Ideally we get rid of the driver-constructor all the way. There's no point in bundling if we split it into sub-systems nicely later. For now it's fine to linearize.